src/components/secondaryHeadline/SecondaryHeadline.jsx

```diff
@@ -0,0 +1,23 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import "./secondaryHeadline.less";
+
+class SecondaHeadline extends React.Component {
+	render() {
+		return (
+			<h1 className = 'component-secondary-headline'>
+				{this.props.text}
+			</h1>
```

Comment:
Na stránce by neměl být více než jeden element h1 ... není to dobré z pohledu SEO ... toto už by měla být h2